Signal/src/view controllers/CodeVerificationViewController.m

```diff
@@ -55,30 +67,165 @@ - (instancetype)init
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+    
+    [self createViews];
+    
     [self initializeKeyboardHandlers];
-    _headerLabel.text               = NSLocalizedString(@"VERIFICATION_HEADER", @"Navigation title in the registration flow - during the sms code verification process.");
+}
+
+- (void)createViews {
+    self.view.backgroundColor = [UIColor whiteColor];
+    self.view.opaque = YES;
+
+    // TODO: Move this to UIColor+OWS?
+    UIColor *signalBlueColor = [UIColor colorWithRed:0.1135657504
+                                               green:0.4787300229
+                                                blue:0.89595204589999999
+                                               alpha:1.];
+
+    UIView *header = [UIView new];
+    header.backgroundColor = signalBlueColor;
+    [self.view addSubview:header];
+    [header autoPinWidthToSuperview];
+    [header autoPinEdgeToSuperviewEdge:ALEdgeTop];
+    [header autoSetDimension:ALDimensionHeight toSize:ScaleFromIPhone5To7Plus(60, 60)];
```

Comment:
As I write/rewrite a view, I'll generally be testing it out in the smallest form factor since that's the place where the layout is most difficult to get right.  Then, I'll test is on our largest form factor and tweak the layout to take advantage of the extra space.  As I do the first pass, I'll sprinkle these "scaling" functions as a reminder of values that I should revisit larger on the larger form factor.  I'll then remove them during the second pass for values that I feel like are okay as constants.  

In this case, I never quite felt 100% satisfied with the large form factor layout and was considering revisiting it. But, upon reflection, I decided to commit to how it stands.  Thanks for the prodding.